### PR TITLE
Allow click-to-ping, quote & emoji picker for edit composer too

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/12740

https://user-images.githubusercontent.com/2403652/114567127-d8a39e00-9c6a-11eb-93f0-614a33a241a8.mov

### CODE REVIEW

File: src/dispatcher/actions.ts [hunk 1]
```diff
@@ -138,4 +138,9 @@ export enum Action {
      * Fired when an upload is cancelled by the user. Should be used with UploadCanceledPayload.
      */
     UploadCanceled = "upload_canceled",
+
+    /**
+     * Inserts content into the active composer. Should be used with ComposerInsertPayload
+     */
+    ComposerInsert = "composer_insert",
```
Comment (turt2live) [inline src/dispatcher/actions.ts:145]: it's largely pointless to define this if we also aren't using it

**Review by turt2live — COMMENTED** [review]
overall seems fine - I assume this has been implicitly reviewed by design?

File: src/components/views/context_menus/MessageContextMenu.js [hunk 2]
```diff
@@ -31,6 +31,8 @@ import { isContentActionable } from '../../../utils/EventUtils';
 import {MenuItem} from "../../structures/ContextMenu";
 import {EventType} from "matrix-js-sdk/src/@types/event";
 import {replaceableComponent} from "../../../utils/replaceableComponent";
+import {ComposerInsertPayload} from "../../../dispatcher/payloads/ComposerInsertPayload";
+import {Action} from "../../../dispatcher/actions";
```
Comment (turt2live) [inline src/components/views/context_menus/MessageContextMenu.js:35]: new rules :) 

`{ words }` please (including existing, if you're up for it)

Comment (turt2live) [reply src/components/views/context_menus/MessageContextMenu.js:35]: applies throughout

**Review by turt2live — APPROVED** [review]
otherwise lgtm

**Comment by turt2live** [comment]
also merge conflicts

**Review by nadonomy — APPROVED** [review]

**Pull request merged**